### PR TITLE
Fix blog and FAQ pages

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
This PR brings updated styles for the Blog and FAQ pages.
Updated layout for the mobile devices.

### CODE REVIEW

File: _sass/modules/_buttons.scss [hunk 1]
```diff
@@ -148,3 +149,48 @@
...
+      height: $icon-size--xs;
+      vertical-align: middle;
+      margin: 0 8px 0 0;
+      opacity: .38;
+    }
+
+    &:hover, &:focus {
+      &:before {
+        background: $mainBrandColor;
+        -webkit-mask-image: url('../img/icon-arrow-back.svg');
```
Comment (andreySadomovskiy) [inline _sass/modules/_buttons.scss]: Low support for  `mask-image` property. For example in IE we will have wrong hover state.

image

Comment (andreySadomovskiy) [reply _sass/modules/_buttons.scss]: Please use [border technique](https://www.w3schools.com/howto/howto_css_arrows.asp) or some another.

File: _sass/modules/_nav.scss [hunk 2]
```diff
@@ -104,13 +104,23 @@
 
 // Expandy nav items
 .arrow-r::before {
-  content: "+";
-  margin-right: 8px;
+  content: '';
+  position: absolute;
+  background: url('../img/icon-caret-right.svg') no-repeat center/cover;
+  width: 24px;
```
Comment (andreySadomovskiy) [inline _sass/modules/_nav.scss]: As I remember we have variable for this icon size. Use it , please.

File: _sass/pages/_faq.scss [hunk 3]
```diff
@@ -24,15 +30,42 @@
...
+  .list-unstyled {
+    li {
+      &:hover {
+        h6 {
+          color: $black;
+
+          &.arrow-d,
+          &.arrow-r {
+            &:before {
+              -webkit-transition: all .25s ease-in-out;
```
Comment (andreySadomovskiy) [inline _sass/pages/_faq.scss]: You can add `@mixin` for this kind of transitions to not repeat same properties each time.

**Comment by andreySadomovskiy** [comment]
Could you update this part as well? As we did it for icons on About page.
``` 
@mixin social-icon($image-url) {
    &::before {
      content: '';
      display: inline-block;
      background: url($image-url) no-repeat center/cover;
      width: 35px;
      height: 35px;
      vertical-align: middle;
      margin: 0 8px 0 0;
      opacity: .38;
    }
  }

  .github {
    @include social-icon('../img/icon_github.svg');
  }

  .google {
    @include social-icon('../img/icon_google-plus.svg');
  }

  .twitter {
    @include social-icon('../img/icon_twitter.svg');
  }

  .stack-overflow {
    @include social-icon('../img/icon_stack-overflow.svg');
  }

  .youtube {
    @include social-icon('../img/icon_youtube.svg');
  }
```

**Comment by andreySadomovskiy** [comment]
@JuliaEvseeva , please review my comments.

File: _sass/modules/_nav.scss (hunk 2 above)
Comment (JuliaEvseeva) [reply _sass/modules/_nav.scss]: Done

File: _sass/pages/_faq.scss (hunk 3 above)
Comment (JuliaEvseeva) [reply _sass/pages/_faq.scss]: Done

File: _sass/modules/_buttons.scss (hunk 1 above)
Comment (JuliaEvseeva) [reply _sass/modules/_buttons.scss]: Done

**Comment by JuliaEvseeva** [comment]
@alexander-yevsyukov, PTAL

**Pull request merged**